### ISSUE TITLE
Remove use of deprecated code

### ISSUE THREAD

**Issue by KnisterPeter** [xing/hops#1098]
In the `hops-apollo-mock-server` in the [middleware](https://github.com/xing/hops/blob/master/packages/apollo-mock-server/lib/mock-server-middleware.js#L4) `hops-config` is used.
This results in a deprecated printed to using projects which could not be worked around. We should remove/update this code so we are on a fully supported track again.

**Comment by robertkowalski** [comment]
fixed solved via 7498871cfd0cd12f2fbbaa211caf9c9ae8bfd68d

verified locally:

```
my-hops-app:info bundling 'graphql-mock-server' finished after 1.6s (2.1 MB)
- hops-graphql-mock-server.js (2.1 MB)
my-hops-app:info bundling 'node' finished after 2.2s (3.57 MB)
```

**Comment by robertkowalski** [comment]
@KnisterPeter can you close the issue?
